lenovo/thinkpad/z/gen2/z13/default.nix

```diff
@@ -5,15 +5,5 @@
     ../../../../../lenovo/thinkpad/z/gen2
   ];
 
-  sound.extraConfig = ''
-    pcm.!default {
-        type plug
-        slave.pcm "hw:1,0"
-    }
-
-    ctl.!default {
-        type hw
-        card 1
-    }
-  '';
+  environment.etc."asound.conf".source = ./asound.conf;
```

Comment:
Per documentation, it says:

> When setting an option, use `lib.mkDefault`.

https://github.com/NixOS/nixos-hardware/blob/master/CONTRIBUTING.md#1-writing-profiles

Did you consider this?